scripts/check_any_file_changed.py

```diff
@@ -0,0 +1,20 @@
+import json
+import os
+import sys
+
+import requests
+
+with open(os.environ["GITHUB_EVENT_PATH"], mode="r", encoding="utf-8") as f:
```

Comment:
It's good to use `if __name__ == "__main__"`

Reply:
done